tensorrt_llm/_torch/peft/lora/layer.py

```diff
@@ -126,7 +126,7 @@ def forward(self, x, lora_params: Dict,
                     True,  # transB
                     max([r.max() for r in lora_ranks]),
                     0,
-                    lora_params["remove_input_padding"],
+                    True,  # TODO smor- should be? lora_params["remove_input_padding"],
```

Comment:
Should be `lora_params["remove_input_padding"]`, but also need to consider that in the lora op.